AdvancedApplications/ODBCApplication/README.md

```diff
@@ -1,20 +1,29 @@
 
-This assumes that you have set up the environment to run a ODBC/DB2CLI application. If you haven't done so, [please follow the instructions here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/tree/master/AdvancedApplications#odbcdb2cli-setup) to set up your environment. 
+If you are not running in the Docker container, run the script to configure SSL which is used when setting up the container which can be found [here](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/blob/master/container/setup/setup-ssl.sh). The script downloads the SSL server certificate through the REST API. This is necessary as Db2 Event Store is configured with SSL with a default keystore out of the box (this may change if you configure your own keystore after installation). Also the following environment variables must be set up
 
-If not running within the docker container, run the script to configure SSL that is provided in the container [set up folder in this repository](https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/blob/master/container/setup/setup-ssl.sh), which will download the SSL server certificate through the rest API. This is necessary because Db2 Event Store is configured with SSL with a default keystore out of the box (this may change if you configure your own keystore after installation). Also set up the environment variables IP with the cluster IP address, EVENT_USER with the user name, and EVENT_PASSWORD with the user password.
+* IP with the cluster IP address
+* EVENT_USER with the user name
+* EVENT_PASSWORD with the user password
 
-### To run the example app [`ExampleODBCApp.c`](ExampleODBCApp.c), follow these steps:
-1. Compile the app by running [`./bldExampleODBCApp.c`](bldExampleODBCApp)`<odbc_client_path>`. The `<odbc_client_path>` is where you have unpacked your ODBC client at the environment setup stage.
-    * The script will then do the following for you:
-       * Find the ODBC headers under `<odbc_client_path>/include` and compile the main app [`ExampleODBCApp.c`](ExampleODBCApp.c) and a helper [`utilcli.c`](utilcli.c) which will check for errors and return the diagnostic messages
-       * Link the db2 library under `<odbc_client_path>/lib`.
-2. Run the executable `./ExampleODBCApp`
+## Running the example app ExampleODBCApp.c
 
-### To clean up the compiler generated files, run:
-[`./bldExampleODBCApp.c`](bldExampleODBCApp) --clean
+1. Compile the app by running
+
+`./bldExampleODBCApp <odbc_client_path>` 
+The `<odbc_client_path>` is where you unpacked the ODBC client during the environment setup. The script does the following:
+     * Finds the ODBC headers under `<odbc_client_path>/include` and compiles the main app ExampleODBCApp.c and a helper utilcli.c which checks for errors and returns diagnostic messages
+     * Links the db2 library under <odbc_client_path>/lib
+     
+2. Run the executable creaed by the build
+
+`./ExampleODBCApp`
```

Comment:
maybe we should move the mentioning of the environment variables this app depends on here instead ? I mean 

* IP with the cluster IP address
* DB2_PORT with the Db2 port number to connect to on the IP address above
* EVENT_USER with the user name
* EVENT_PASSWORD with the user password
* SERVER_CERT_PATH with the path to the downloaded SSL certificate

Reply:
This list is not complete either, you need `DEPLOYMENT_ID` as well.

We should consider a script users can fill in and execute wherever they like with <fill_me> options.  I think it's easier then just stating all the env required to set in various docs i.e. it would just say ... configure and run `. <env_script>`

Reply:
actually DEPLOYMENT_ID is only required by the setup SSL script, it is not required by the app. But I do agree it would be better to have a configure app that would be used to set the environment.

Reply:
Yeah I meant to leave this comment further above when we were talking about required env vars for the script.

Either way it'll probably be easier just to have them all set and explain them just one (in said script too), other solutions tend to take an approach like that from what I've seen.

Reply:
I ran the ODBC without setting the variables and it connected, and created the table, but failed on the inserts. I will clean it up to make sure its clear that they need to be set prior to running the applicaiton. 

 Just ran with the setting the variables and it went to completion, i did not set the server cert path (if that was to be set should i include instructions on how to get it?)

Reply:
@robertpangborn if you are running within the docker container, all of these are already set in the environment, as you are already passing these as part of the arguments passed to dockershell, which results in them being set in the container environment. The goal of this is to describe in words what are those requirements, so that people understand the pre-requisites, as we are describing here both ways of running these: 1) whitin the docker container, and 2) outside the docker container.  

As for what sets up the server cert path, it is the setup_ssl.sh script, it adds it to the .bashrc that will be set each time the user enters the container, here https://github.com/IBMProjectEventStore/db2eventstore-IoT-Analytics/blob/master/container/setup/setup-ssl.sh#L121